ring/model.go

```diff
@@ -21,6 +21,13 @@ func (ts ByAddr) Len() int           { return len(ts) }
 func (ts ByAddr) Swap(i, j int)      { ts[i], ts[j] = ts[j], ts[i] }
 func (ts ByAddr) Less(i, j int) bool { return ts[i].Addr < ts[j].Addr }
 
+// ByID is a sortable list of InstanceDesc.
```

Comment:
I've left `ByAddr` because it is used [outside of the library](https://github.com/search?q=org%3Agrafana++ring.ByAddr&type=code)